### PR TITLE
#8633: Update ttnn.full, ttnn.ones, ttnn.zeros documentation

### PR DESCRIPTION
**Pull request by VirdhatchaniKN**
Issue : #8633

### What's changed

- Updating docs based on [#8633-comment](https://github.com/tenstorrent/tt-metal/issues/8633#issuecomment-2371082993) for `ttnn.full`, `ttnn.ones`, `ttnn.zeros`.
- Test File : `tests/ttnn/python_api_testing/non_working_unit_tests/wormhole/test_eltwise_full_and_ones.py` - PASSED

[CI](https://github.com/tenstorrent/tt-metal/actions/runs/11347905619) : PASSED

### CODE REVIEW

**Review by mcw-anasuya — APPROVED** [review]

File: tests/ttnn/python_api_testing/non_working_unit_tests/wormhole/test_eltwise_full_and_ones.py [hunk 1]
```diff
@@ -69,7 +69,7 @@ def run_ones_tests(input_shape, dtype, dlayout, in_mem_config, output_mem_config
 
 test_sweep_args = [
     (
-        [(4, 7, 21, 133)],
+        [(4, 7, 32, 32)],
```
Comment (KalaivaniMCW) [inline tests/ttnn/python_api_testing/non_working_unit_tests/wormhole/test_eltwise_full_and_ones.py:72]: 32, 160 (nearest multiples) ?

Comment (Aswinmcw) [reply tests/ttnn/python_api_testing/non_working_unit_tests/wormhole/test_eltwise_full_and_ones.py:72]: +1

**Review by Aswinmcw — APPROVED** [review]
LGTM

**Review by mouliraj-mcw — APPROVED** [review]

File: tests/ttnn/python_api_testing/non_working_unit_tests/wormhole/test_eltwise_full_and_ones.py (hunk 1 above)
Comment (VirdhatchaniKN) [reply tests/ttnn/python_api_testing/non_working_unit_tests/wormhole/test_eltwise_full_and_ones.py:72]: Sure

**Review by ayerofieiev-tt — APPROVED** [review]

**Comment by VirdhatchaniKN** [comment]
https://github.com/tenstorrent/tt-metal/actions/runs/11358103764 - passed

**Pull request merged**